change/@fluentui-react-spinner-65e1fe2a-ca72-408e-90eb-6a1ca3f6caca.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30567&commitId=ab40872c05021fee2594caaf483029cd2f7bc30a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Spinner converged</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Inverted with Label.chromium.png|577| Changed|
|Spinner converged.Primary + size.chromium.png|935| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|577| Changed|
|Spinner converged.Inverted.chromium.png|144| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|1108| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|144| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|272| Changed|
|Spinner converged.Primary with Label.chromium.png|523| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|89| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|277| Changed|
|Spinner converged.Primary.chromium.png|131| Changed|
|Spinner converged.Primary with Label - RTL.chromium.png|523| Changed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|357| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|1088| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
The old VR tests mistakenly set the rotation speed twice as fast as it should be with the old SVG animation, but are the correct speed as of this PR. As a result, the rotation is different in the screenshots. It's just a bug with the VR tests, and fixed in this PR.

Otherwise, there are some pixel differences, but they are relatively minor.

Reply:
VR tests are fixed now. There are some slight differences due to the different way of rendering an animating the spinner, but they are expected, and look ok to me.

Reply:
@spmonahan pointed out that the Inverted High Contrast visuals have changed so that the track is no longer visible. The track was never visible in the normal High Contrast visuals, so now Inverted matches Normal. 

This change is a result of using a token for the background color instead of a hardcoded value. If there is a bug, it will need to be fixed on the tokens. However, I'm not going to block this PR on it because the normal (non-inverted) spinner never had a visible track in high contrast. Now the inverted spinner matches that.

Additionally, the "High Contrast" VR test is using the high contrast web theme; not the OS high contrast mode (aka "forced colors"). Our VR tests don't cover OS high contrast mode. However, I've manually checked and verified that the track is visible in Windows high contrast mode.

Reply:
Sounds good to me. Do we have an issue to follow up on 

> If there is a bug, it will need to be fixed on the tokens

Reply:
I double checked; this is by design. The colors in this PR match the design spec:

![screenshot of Spinner design spec showing colors](https://github.com/microsoft/fluentui/assets/48106640/7964cad3-5787-4329-973a-1d9a3be2de35)